Task4/src/test/java/by/innowise/tasks/repository/SubjectRepositoryTest.java

```diff
@@ -39,7 +39,7 @@ public void findSubjectById() {
 
     @Test
     public void getAllSubjects() {
-        assertFalse(subjectRepository.findAll().isEmpty());
+        assertTrue(subjectRepository.getAll().findFirst().isPresent());
```

Comment:
we have to compare this list with emptyList. Use Collections.emptyList() in order to get empty list